.github/workflows/continuous-deployment.yml

```diff
@@ -16,16 +16,16 @@ on:
         default: 'false'
         type: choice
         options:
-        - 'false'
-        - 'true'
+          - 'false'
```

Comment:
Notice mostly formatting updates to this file. Is there a linter or formatter that the team could use?